web/src/index.css

```diff
@@ -25,3 +25,9 @@ code {
 .ant-table.ant-table-middle .ant-table-title, .ant-table.ant-table-middle .ant-table-footer, .ant-table.ant-table-middle thead > tr > th, .ant-table.ant-table-middle tbody > tr > td {
   padding: 1px 8px !important;
 }
+
+.ant-layout-footer {
+  bottom: 0;
+  position: fixed;
```

Comment:
I think this is not the right way to fix that issue , this will cause other item to hide behind the footer, someone already filed a pr for the same issue and @RoLoDeXx requested some changes on that pr too. 
Humble request please check the similar issue and pr related to that before creating a new.

Reply:
Just noticed the pr, will close it right away. 

Also, wouldn't using custom css / inline styling (as is done in the other pr you are talking about) incur the same problem you're mentioning above?